package.json

```diff
@@ -32,8 +32,7 @@
     "eslint": "^2.7.0",
     "eslint-config-loopback": "^1.0.0",
     "express": "^4.12.0",
-    "loopback": "2.x.x",
-    "loopback-datasource-juggler": "2.x.x",
```

Comment:
You are removing `loopback-datasource-juggler` but many tests have `require('loopback-datasource-juggler')` in them.